ghost/core/test/e2e-api/admin/pages.test.js

```
@@ -1,7 +1,8 @@
 const {mobiledocToLexical} = require('@tryghost/kg-converters');
 const models = require('../../../core/server/models');
-const {agentProvider, fixtureManager, mockManager, matchers} = require('../../utils/e2e-framework');
-const {anyArray, anyBoolean, anyContentVersion, anyEtag, anyLocationFor, anyObject, anyObjectId, anyISODateTime, anyString, anyUuid} = matchers;
+const {agentProvider, fixtureManager, mockManager, matchers, assertions} = require('../../utils/e2e-framework');
+const {anyArray, anyBoolean, anyContentVersion, anyEtag, anyLocationFor, anyObject, anyObjectId, anyISODateTime, anyString, anyUuid, stringMatching} = matchers;
+const {cacheInvalidateHeaderSetToWildcard} = assertions;
```

ESLint

[error] 'cacheInvalidateHeaderSetToWildcard' is assigned a value but never used. Allowed unused vars must match /^should$/u.

(no-unused-vars)